staging/src/k8s.io/client-go/tools/cache/index.go

```diff
@@ -75,6 +75,7 @@ func IndexFuncToKeyFuncAdapter(indexFunc IndexFunc) KeyFunc {
 }
 
 const (
+	// NamespaceIndex is the most popular name to a IndexFunc.
```

Comment:
"popular name" seems odd.

"NamespaceIndex" is the lookup name for the most comment index function, which is to index by the namespace field"?

Reply:
Done. Thanks.